Add Exercises from Ververica's troubleshooting training (#1)
ververica/flink-training: troubleshooting/introduction/README.md

| Fake Kafka Source | --> | Watermarks/Timestamps | --> | Deserialization | --> | Windowed Aggregation | --> | Sink: NormalOutput |
|                   |     |                       |     |                 |     |                      |     |                    |
+-------------------+     +-----------------------+     +-----------------+     +----------------------+     +--------------------+
                                                                                            \
                                                                                             \               +--------------------+
                                                                                              \              |                    |
                                                                                               +-----------> | Sink: LateDataSink |    
                                                                                                             |                    |
                                                                                                             +--------------------+
```

> In local mode, sinks print their values on `stdout` (NormalOutput) and `stderr` (LateDataSink) for simplified debugging
> while as without local mode, a `DiscardingSink` is used for each sink.

Reviewer: I think this is easier to understand if it's two sentences. Also "while as" isn't correct English. Perhaps you meant "whereas".

```suggestion
In local mode, the sinks print their values on `stdout` (NormalOutput) and `stderr` (LateDataSink), which simplifies debugging.
Otherwise, a `DiscardingSink` is used for each sink.
```
